test/node/reducers/vis-state-test.js

```diff
@@ -384,7 +384,7 @@ test('#visStateReducer -> LAYER_TYPE_CHANGE.2', t => {
   );
   const newLayer2 = nextState2.layers[0];
   t.equal(newLayer2.config.sizeField, intField, 'should update sizeField');
-  t.equal(newLayer2.config.sizeScale, 'sqrt', 'should scale to linear');
+  t.equal(newLayer2.config.sizeScale, 'sqrt', 'should scale to sqrt');
```

Comment:
Not sure why this changed @heshan0131 ....

Reply:
the comment in test is probably copied.